zap/zap.gradle.kts

```diff
@@ -41,56 +41,56 @@ tasks.named<JacocoReport>("jacocoTestReport") {
 }
 
 dependencies {
-    api("com.fifesoft:rsyntaxtextarea:3.0.4")
+    api("com.fifesoft:rsyntaxtextarea:3.1.1")
     api("com.github.zafarkhaja:java-semver:0.9.0")
     api("commons-beanutils:commons-beanutils:1.9.4")
-    api("commons-codec:commons-codec:1.13")
-    api("commons-collections:commons-collections:3.2.2")
+    api("commons-codec:commons-codec:1.15")
     api("commons-configuration:commons-configuration:1.10")
     api("commons-httpclient:commons-httpclient:3.1")
-    api("commons-io:commons-io:2.6")
+    api("commons-io:commons-io:2.8.0")
     api("commons-lang:commons-lang:2.6")
-    api("org.apache.commons:commons-lang3:3.9")
-    api("org.apache.commons:commons-text:1.8")
+    api("org.apache.commons:commons-lang3:3.11")
+    api("org.apache.commons:commons-text:1.9")
     api("edu.umass.cs.benchlab:harlib:1.1.2")
     api("javax.help:javahelp:2.0.05")
     api("log4j:log4j:1.2.17")
     api("net.htmlparser.jericho:jericho-html:3.4")
     api("net.sf.json-lib:json-lib:2.4:jdk15")
-    api("org.apache.commons:commons-csv:1.7")
-    api("org.bouncycastle:bcmail-jdk15on:1.64")
-    api("org.bouncycastle:bcprov-jdk15on:1.64")
-    api("org.bouncycastle:bcpkix-jdk15on:1.64")
-    api("org.hsqldb:hsqldb:2.5.0")
-    api("org.jfree:jfreechart:1.0.19")
-    api("org.jgrapht:jgrapht-core:0.9.0")
+    api("org.apache.commons:commons-collections4:4.4")
```

Comment:
We should not introduce it as `api`, it does not seem to be needed though.

Reply:
Reverted.